live-site/components/judging-section/judging-schedule-judges/cell/Cell.styled.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { max } from '../../../../../shared-ui/lib/responsive';
+import { colors } from '../../../../../shared-ui/style/colors';
+import { fonts } from '../../../../../shared-ui/style/typography';
+
+const StyledCell = styled.div`
+background-color: ${colors.WHITE};
+width: 4.5em;
+height: 1.5em;
+border: 0.02em solid ${colors.TEXT_BROWN};
+text-align: center;
+font-family: ${fonts.nunitoSansRegular};
+font-size: 1.8em;
```

Comment:
I think we can make the font-size a bit smaller like 1.5em in case there's a lot of content